docs: update blog contribution guidelines (#798)
iresty/iresty.github.io: website/docs/general/blog-contributing-guide.md

#### Blog's header information

##### Single author template

Our blogs start with a header sections containing fields such as `title`, `author`, `authorURL`, `authorImageURL`, `keywords`, `description`, and `tags`.

> While `title` and `author` are starightforward, the other fields might be a bit confusing for first-time contributors. Explanations of each field is shown below. Please note that these fields could change over time.

Reviewer: ```suggestion
While `title` and `author` are starightforward, the other fields might be a bit confusing for first-time contributors. Explanation of each field is mentioned below. Please note that these fields could change over time.
```

The author's GitHub avatar, it is related with `authorURL`. For the same reason described  above, this is an optional field. See below for how to get the avatar.

#### authors

> Required field for co-author template. When a blog is co-authored by 2 people, please use `authors` instead of `author` to give credits to both authors. `authors` consists of the following fields: `authors.name`, `authors. url`, and `authors.image_url.`

Reviewer 2: ```suggestion
The required field for co-author template. When a blog is co-authored by 2 people, please use `authors` instead of `author` to give credits to both authors. `authors` consists of the following fields: `authors.name`, `authors.url`, and `authors.image_url.`
```

- `authors.name`: authors' names in plain text, for example: `name: "John Doe"`.

> - `authors. url`: authors' GitHub pages, for example: `url: "www.google.com"`.

Reviewer 2: ```suggestion
- `authors.url`: authors' GitHub pages, for example: `url: "www.google.com"`.
```

> The author's GitHub avatar, it is related with `authorURL`. For the same reason described  above, this is an optional field. See below for how to get the avatar.

Reviewer: ```suggestion
The author's GitHub avatar, it is related with `authorURL`. For the same reason described above, this is an optional field. See below for how to get the avatar.
```

We can help authors to use a better category once a pull request is submitted.

#### How to get the authorImageURL or authors.image_url

> 1. Open Chrome or your preffered browser.

Reviewer: ```suggestion
1. Open Chrome or your preferred browser.
```
